deploy/ray-operator/controllers/common/constant.go

```diff
@@ -0,0 +1,27 @@
+package common
+
+const (
+	// use as pod type decision
+	Head = "head"
+
+	// use as label key
```

Comment:
label key for what?

Reply:
label key mainly for selecting the pod, distinguish pod by cluster name.
- ClusterPodType for selecting head or worker
- rayOperator for selecting the pod created by ray-operator
- RayClusterOwnerKey  for selecting pod created by which cluster
...